modules/cmdrunner/widget.go

```diff
@@ -121,10 +123,13 @@ func runCommandLoop(widget *Widget) {
 		<-widget.runChan
 		widget.resetBuffer()
 		cmd := exec.Command(widget.settings.cmd, widget.settings.args...)
-		cmd.Stdout = widget
 		cmd.Env = widget.environment()
-		err := cmd.Run()
+		f, err := pty.Start(cmd)
+		if err != nil {
+			panic(err)
```

Comment:
I don't think we should panic here. If the `pty` fails for some reason, we should fall back to the old way to still provide command output.

Reply:
Thanks. Major brainfart on my behalf.